cloud/scope/cluster.go

```diff
@@ -46,22 +46,22 @@ func NewClusterScope(params ClusterScopeParams) (*ClusterScope, error) {
 	if params.Cluster == nil {
 		return nil, errors.New("failed to generate new scope from nil Cluster")
 	}
-	if params.AzureCluster == nil {
-		return nil, errors.New("failed to generate new scope from nil AzureCluster")
+	if params.AzureStackEdgeCluster == nil {
+		return nil, errors.New("failed to generate new scope from nil AzureStackEdgeCluster")
 	}
 
 	if params.Logger == nil {
 		params.Logger = klogr.New()
 	}
 
 	log := klogr.New()
-	err := params.AzureClients.setDBECredentials(params.AzureCluster.Spec.SubscriptionID)
+	err := params.AzureClients.setDBECredentials(params.AzureStackEdgeCluster.Spec.SubscriptionID)
```

Comment:
setDBECredentials --> should we change DBE to AzureStackEdge to be consistent

Reply:
Renamed setDBECredentials  to setAzureStackEdgeCredentials in the latest commit